code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -94,7 +94,23 @@
 				new /obj/item/weapon/kitchen/utensil/knife/large/butch/meatcleaver(src)
 				new /obj/item/weapon/storage/pill_bottle/hyperzine(src)
 				new /obj/item/weapon/card/id/syndicate(src)
-
+				
+			if("cowboy")
+				new /obj/item/clothing/accessory/storage/bandolier(src)
+					contents = list()
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
+					new /obj/item/ammo_storage/speedloader/a357(src)
```

Comment:
6 speedloaders, plus the rounds in the mateba, makes 42 rounds of ammunition for a gun that can 2-tap a guy

Holy crap